react/SlideToggle/SlideToggle.less

```diff
@@ -50,7 +51,8 @@
 
 .label {
   padding: 0;
-  display: inline;
+  line-height: @grid-row-height * 3;
+  transform: none;
```

Comment:
@michaeltaranto any more thoughts on this?  I guess the problem is that we currently don't have a concept of responsive text without baseline correction.  Some options I can think of are:

1.  Use the `standardTextResponsive` mixin without the text component.  This would still require overriding basekick.
2.  Change the responsive type mixins so they accept a param to turn off basekick.
3.  Use `touchableText`.  What about this?

```
.label {
  .touchableText();
  margin-top: -@grid-row-height;

  @media only screen and (min-width: @responsive-breakpoint) {
    .touchableText(@standard-type-scale);
  }
}
```